cmd/kubeadm/app/cmd/join.go

```diff
@@ -34,9 +34,9 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/discovery"
 	kubenode "k8s.io/kubernetes/cmd/kubeadm/app/node"
-	kubeconfigphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubeconfig"
 	"k8s.io/kubernetes/cmd/kubeadm/app/preflight"
 	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
```

Comment:
I actually think it's much better to throw `util`  at the end so one knows for sure that it's question about a generic helpers lib.

If the name was kubeconfig, I'd think it the question was about the kubeconfig phase for example, or a k8s-generic factory or something